src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -19,7 +18,6 @@ export class ConfirmationPage extends React.Component {
   componentDidMount() {
     focusElement('.schemaform-title > h1');
```

Comment:
You can use our reusable module for this:
```js
import { focusElement } from 'platform/utilities/ui';
```

I think we may be missing an import for this function 🙂